styles/tables/mod.css

```diff
@@ -1,44 +1,59 @@
+/* Modernized table styles */
+
 table {
-  display: block;
+  width: 100%;
   max-width: 100%;
   margin: 0 auto 1rem;
   border-collapse: collapse;
-  border-spacing: 0;
-  inline-size: fit-content;
   overflow-x: auto;
+  border-radius: 0.5rem;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
 }
 
 .table-responsive {
-  display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(0, 1fr));
+  display: block;
+  width: 100%;
   overflow-x: auto;
 }
 
-.table-responsive table {
-  display: table;
-}
-
 caption {
-  margin-top: .5rem;
+  margin-top: 0.5rem;
   caption-side: bottom;
   color: var(--muted);
+  font-size: 0.875rem;
```

Comment:
Not sure if decreasing font-size on a already muted element is really needed
Especially with the larger padding it looks disproportionate:
![image](https://github.com/lowlighter/matcha/assets/22963968/64070417-b673-443a-89f9-c12bc9ff4f2f)